Services/CO.CDP.Organisation.WebApi/Api/Organisation.cs

```diff
@@ -101,6 +101,31 @@ await useCase.Execute(organisationId)
                return operation;
            });
 
+        app.MapGet("/organisations/{organisationId}/reviews",
+            [OrganisationAuthorize(
+                [AuthenticationChannel.OneLogin, AuthenticationChannel.ServiceKey],
+                [Constants.OrganisationPersonScope.Admin, Constants.OrganisationPersonScope.Editor, Constants.OrganisationPersonScope.Viewer],
+                OrganisationIdLocation.Path,
+                [Constants.PersonScope.SupportAdmin])]
+        async (Guid organisationId, IUseCase<Guid, IEnumerable<Review>> useCase) =>
+        await useCase.Execute(organisationId)
+                   .AndThen(organisation => organisation != null ? Results.Ok(organisation) : Results.NotFound()))
```

Comment:
Copy&Paste problem. This isn't an organisation we're dealing with here :D

Reply:
Done